pkg/volume/cinder/attacher.go

```diff
@@ -278,6 +278,15 @@ func (detacher *cinderDiskDetacher) Detach(deviceMountPath string, nodeName type
 		instanceid = instanceid[(ind + 1):]
 	}
 
+	pending, volumeStatus, err := detacher.cinderProvider.OperationPending(volumeID)
+	if err != nil {
+		// Continue attach, the same as DiskIsAttached()
```

Comment:
Yes, the same as DiskIsAttached().
If OperationPending() return err, DiskIsAttached() will return same err, we can ignore it and try detach anyway.